src/Symfony/Component/Console/Helper/Helper.php

```diff
@@ -72,6 +73,18 @@ public static function substr($string, $from, $length = null)
         return mb_substr($string, $from, $length, $encoding);
     }
 
+    /**
+     * Replaces text within a portion of a string.
+     */
+    public static function substr_replace(string $string, string $replacement, int $start, int $length = null): string
+    {
+        if (false === mb_detect_encoding($string, null, true)) {
+            return substr_replace($string, $replacement, $start, $length);
+        }
+
+        return mb_substr($string, 0, $start).$replacement.mb_substr($string, $start + $length);
```

Comment:
from http://php.net/manual/en/function.substr-replace.php

i think we should default `$length = strlen($string);` here as well

Reply:
That looks working as expected https://3v4l.org/C5Nhs, do I miss something ?